ci/test/00_setup_env_native_centos.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2019 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+export LC_ALL=C.UTF-8
+
+export DOCKER_NAME_TAG=centos:7
+export DOCKER_PACKAGES="gcc-c++ libtool make git python3"
+export PACKAGES="boost-devel libevent-devel libdb4-devel libdb4-cxx-devel miniupnpc-devel zeromq-devel qt5-qtbase-devel qt5-qttools-devel qrencode-devel"
```

Comment:
Please install python3-zmq, if available. See https://travis-ci.org/bitcoin/bitcoin/jobs/619963005#L3914

Reply:
Done.

Reply:
And https://apps.fedoraproject.org/packages/python3-zmq

Reply:
https://centos.pkgs.org/7/epel-x86_64/python36-zmq-14.7.0-8.el7.x86_64.rpm.html

Reply:
Ah, please add it then.

Reply:
Already:

https://github.com/bitcoin/bitcoin/blob/7f3ae224685efaeb6fe714de90e8871d12e55f34/ci/test/00_setup_env_native_centos.sh#L10

Reply:
https://travis-ci.org/bitcoin/bitcoin/jobs/621765674